electron/anticheat/utils.ts

```diff
@@ -0,0 +1,37 @@
+import { heroicAnticheatDataPath, isLinux } from '../constants'
+import * as axios from 'axios'
+import { logInfo, LogPrefix, logWarning } from '../logger/logger'
+import { readFileSync, writeFileSync } from 'graceful-fs'
+import { AntiCheatInfo } from '../types'
+import { epicTitleToAnticheat } from './titles_map'
+
+async function downloadAntiCheatData() {
+  if (!isLinux) return
```

Comment:
Does this mean we only show the info on Linux? Because I guess most of those titles are not available for macOS natively so we should show them on mac as well.

Reply:
hmmmm maybe we should have a different source if on a Mac? the site says `A comprehensive and crowd-sourced list of games using anti-cheats and their compatibility with GNU/Linux or Wine/Proton.` so I thought it was linux specific, like... I understand Fortnite works on a Mac so this would report an incorrect alert in a case like that?

Reply:
maybe I have to only check this if not a native mac version of a game when we detect it's a mac os?

I'm still not sure if areweanticheatyet considers mac games on the status though, the list is built with linux in mind so mac might dispaly false positives/negatives?

Reply:
> maybe I have to only check this if not a native mac version of a game when we detect it's a mac os?
> 
> I'm still not sure if areweanticheatyet considers mac games on the status though, the list is built with linux in mind so mac might dispaly false positives/negatives?

As far as I'm aware if the game is not native on Mac Cross-Over is currently the only option - However I don't think that the EAC/BattlEye runtime will work with that - So it's basically: Every game that doesn't work natively on MacOS and includes intrusive Anti-Cheat like EAC / BattlEye won't work.

Some games that don't use strong client side Anti-Cheat or solely Server-Side AntiCheat, for example Star Wars Battlefront (2) actually run in Cross-Over I believe

So AWACY can theoretically be used to determine if a non native game could work on MacOS, but intrusive Anti-Cheats will most likely not work.